### PR TITLE
MM-1660 Added singlequotes around 'Edit' and did cosmetic refactoring

### PR DESCRIPTION
**Pull request by nickago**
Cosmetic refactoring includes:
   * Double quotes to single
   * Add/Remove spaces around function and objects as dictated
   * Split one line functions into multiple lines

### CODE REVIEW

File: web/react/components/user_settings.jsx [hunk 1]
```diff
@@ -966,57 +984,60 @@ var GeneralTab = React.createClass({
...
+                    }}
                     picture={this.state.picture}
                     pictureChange={this.updatePicture}
                     submitActive={this.submitActive}
                 />
             );
-
-        } else {
-            var minMessage = "Click Edit to upload an image.";
+        }else {
```
Comment (jwilander) [inline web/react/components/user_settings.jsx]: Space between } and else is missing :)

**Comment by crspeller** [comment]
Is this all cosmetic?

**Comment by jwilander** [comment]
No, if you search 'Edit' you'll see that it does update some text slightly

**Comment by crspeller** [comment]
Ahh I was confused by the title because I thought the single quotes where for ESLint. 
@nickago Fix Joram's good spot and we can merge this.

**Comment by nickago** [comment]
@crspeller the change has been made, please merge

**Pull request merged**